test/opcode/concat.lua

```diff
@@ -0,0 +1,52 @@
+--[[----------------------------------------------------------------------------
+ 
+  (c) Copyright IBM Corp. 2017

```

Comment:
Could you make this `(c) Copyright IBM Corp. 2017, 2017`. It will make it easier to write a script to update this later on.